cookbooks/wazuh_filebeat/recipes/filebeat.rb

```diff
@@ -28,7 +28,7 @@
 
 bash 'Import Wazuh module for filebeat' do 
   code <<-EOH
-  curl -s https://packages-dev.wazuh.com/3.x/filebeat/wazuh-filebeat-0.1.tar.gz | tar -xvz -C /usr/share/filebeat/module
+  curl -s https://s3-us-west-1.amazonaws.com/packages.wazuh.com/3.x/filebeat/wazuh-filebeat-0.1.tar.gz | tar -xvz -C /usr/share/filebeat/module
```

Comment:
We can use the standard production link as stated in the documentation:

`https://packages.wazuh.com/3.x/filebeat/wazuh-filebeat-0.1.tar.gz`

Also adding a default attribute to manage `wazuh-filebeat-0.1-.tar.gz` is recommended in order to have a good maintainability

Reply:
Hi @jm404,

You're right that would be the official link to the package.

Here you have the [changes](https://github.com/wazuh/wazuh-chef/commit/1a0a6fb2f4d069ed316f43f989d3a47e74211656).

Kind regards,

Rshad